### PR TITLE
Add some worker metrics

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: src/util/worker/future.rs [hunk 1]
```diff
@@ -75,11 +79,14 @@ where
     R: Runnable<T> + Send + 'static,
     T: Display + Send + 'static,
 {
+    let name = thread::current().name().unwrap().to_owned();
     let mut core = Core::new().unwrap();
     let handle = core.handle();
     {
         let f = rx.take_while(|t| Ok(t.is_some())).for_each(|t| {
             runner.run(t.unwrap(), &handle);
+            WORKER_PENDING_TASK_VEC.with_label_values(&[&name]).sub(1.0);
```
Comment (siddontang) [inline src/util/worker/future.rs]: seem we can introduce a count value, not only we can use gauge set here but also can use it to check is_busy.

**Comment by siddontang** [comment]
Rest LGTM

File: src/util/worker/future.rs (hunk 1 above)
Comment (huachaohuang) [reply src/util/worker/future.rs]: Maybe later, this PR only handles the metrics.

**Comment by huachaohuang** [comment]
PTAL @BusyJay

**Comment by huachaohuang** [comment]
PTAL @zhangjinpeng1987

**Pull request merged**